README.md

```diff
@@ -3,6 +3,8 @@
 ComposeUI
 =========
 
+Feature gaps with respect to UI components, layout management, and subpar native hosting in other industry container solutions pointed to us investing in the creation of this standard container as an open desktop platform. It is to provide a hybrid solution that meets the needs of the diverse application catalogs as well as a compelling opensource alternative.
```

Comment:
> pointed to us

pointed us to ?

>It is to provide a

It will provide a ?

>the needs of the diverse application catalogs

the needs of a diverse application catalog ?